src/cli_shared/snapshot.rs

```diff
@@ -69,10 +68,11 @@ pub async fn fetch(
     chain: &NetworkChain,
     vendor: TrustedVendor,
 ) -> anyhow::Result<PathBuf> {
-    let (_len, url) = peek(vendor, chain).await?;
-    let (date, height, forest_format) = ParsedFilename::parse_url(&url)
-        .context("unexpected url format")?
+    let (_len, path) = peek(vendor, chain).await?;
```

Comment:
Doc comment seems out of sync. It is stating that we try to download first with `aria2C`.

Reply:
Is `aria2c` still need to be a dependency of forest repository?

Reply:
`aria2c` is no longer used at all. I'll update the documentation.